quinn-proto/src/endpoint.rs

```diff
@@ -235,9 +232,7 @@ impl Endpoint {
             };
         }
 
-        //
         // Potentially create a new connection
-        //
```

Comment:
Extremely nitty: where the comment line describes a longer block (especially if the block it describes is itself split up into multiple blocks by empty lines), I prefer to keep an empty line between the comment and the block -- and for consistency, I might prefer keeping it for all of these.

Reply:
It gets a bit strange with the if-else but oh well